util/finding.go

```diff
@@ -83,7 +82,7 @@ func OutputFinding(finding Finding) {
 
 			source := finding.Untrusted_Source[0]
 			fmt.Printf("\n%s:%d\n", source.SourceFilename, source.SourceLineNum)
-			fmt.Printf("Source of Untrusted Input: [ %s ]\n", parentFunctionNoArgs)
+			fmt.Printf("Source of Untrusted Input: [ %s ]\n", source.ParentFunction)
```

Comment:
should we run the same `StripArguments()` function here that we do on the sink function signature? https://github.com/praetorian-inc/gokart/blob/main/util/finding.go#L74

I think we should since this will keep the non-verbose output clean and concise. We can print the full function signatures below in the verbose output.

Reply:
also, since the `parentFunctionNoArgs` variable is now only used once, I think we should just call the `StripArguments()` function in-line on the line of output where it's used. That, or we should rename the variable to something that identifies sink, like `sinkParentFunctionNoArgs`

Reply:
Ok, resolved.  The trace does look much better with the args stripped as well.